src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -168,6 +168,21 @@ struct WeylElectric : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
 
+/*!
+ * \brief Computes the magnetic part of the Weyl tensor in vaccum.
+ * \details Computes the magnetic part of the Weyl tensor \f$B_{ij}\f$
```

Comment:
Since the simple tag doesn't compute anything, I would just delete these \details

Reply:
Should I leave the latex that comes after the details?